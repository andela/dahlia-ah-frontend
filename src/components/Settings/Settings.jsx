import React from 'react';
import './Settings.scss';
import SettingsItem from '../SettingsItem/SettingsItem';

const Settings = () => {
  const user = JSON.parse(window.localStorage.getItem('AuthorsHavenUser')) || { allowEmailNotification: true };
  return (
    <div className="settings">
      <div className="content">
        <h5>Settings</h5>
        <div>
          <SettingsItem
            initialState={user.allowEmailNotification}
            title="Receive email notification"
            description="An email will be sent to you when ever you receive a notification or any other event"
          />
        </div>
      </div>
    </div>
  );
};

export default Settings;
