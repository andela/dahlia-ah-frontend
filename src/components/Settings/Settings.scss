.settings {
  height: calc(100vh - 240px);
  padding: 90px 0 0 0;
  display: flex;
  justify-content: center;
  .content {
    width: 80%;
    h5 {
      font-style: normal;
      font-weight: bold;
      font-size: 22px;
      line-height: 21px;
    }
  }
}