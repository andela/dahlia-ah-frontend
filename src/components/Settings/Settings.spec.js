import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Settings from './Settings';

configure({ adapter: new Adapter() });

describe('Settings component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <Settings />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
