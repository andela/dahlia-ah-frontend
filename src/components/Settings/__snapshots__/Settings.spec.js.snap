// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings component renders correctly 1`] = `ShallowWrapper {}`;
